sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1192,3 +1192,33 @@ def test_dtype_match():
             lr_64.fit(X_64, y_64)
             assert_equal(lr_64.coef_.dtype, X_64.dtype)
             assert_almost_equal(lr_32.coef_, lr_64.coef_.astype(np.float32))
+
+
+def test_warm_start_converge_LR():
+    # Test to see that the logistic regression converges on warm start,
+    # with multi_class='multinomial'
+    # Set independent variable values
+    rng = np.random.RandomState(0)
+    X = np.concatenate((rng.randn(100, 2) + [1, 1], rng.randn(100, 2)))
+    y = [1] * 100 + [-1] * 100
+
+    lr1 = LogisticRegression(C=1000000, multi_class='multinomial',
+                             solver='sag', tol=0.0001, warm_start=False,
+                             verbose=0)
+    lr2 = LogisticRegression(C=1000000, multi_class='multinomial',
+                             solver='sag', tol=0.0001, warm_start=True,
+                             verbose=0)
+    lr1_loss = []
```

Comment:
you can use a list comprehension

lr_no_ws_loss = [log_loss(y, lr_no_ws.fit(X, y).predict_proba(X)) for _ in range(5)]